docs/docs/community/maintainers/triage-issues.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Issue triaging
+description: Guidelines and tips for maintaining Nebari
+---
+
+Issue triage is a process by which Nebari maintainers and contributors intake and reviews new GitHub issues and requests,
+and organize them to be actioned. Triaging involves categorizing issues and Pull Requests based on factors such as priority,
+area of ownership of the issue (tags), and the issue kind (bug, feature, and so on).
+
```

Comment:
What are your thoughts on including a goal for folks?

```suggestion

We aim to triage new issues or Pull Requests within 1-2 working days.
```

Reply:
agree!